modules/ossm-validating-smcp.adoc

```diff
@@ -0,0 +1,103 @@
++
+[NOTE]
+====
+You view the Kiali components under the Kiali Operator, not the {ProductShortName} Operator.
+====
++
+* Calls the Jaeger Operator to create Jaeger components based on configuration in either the SMCP or the Jaeger custom resource.
++
+[NOTE]
+====
+You view the Jaeger components under the Jaeger Operator and the Elasticsearch components under the Elasticsearch Operator, not the {ProductShortName} Operator.
+====
++
+
+
+.From the Openshift console
+
+You can verify the `ServiceMeshControlPlane` installation in the OpenShift Console.
+
+. Navigate to *Operators* -> *InstalledOperators*.
+. Select the `<istio-system>` namespace.
+. Select the {ProductName} Operator.
+. Click the *Istio Service Mesh Control Plane* tab.
+. Click the name of your control plane, for example `basic`.
+. To view the resources created by the deployment, click the *Resources* tab. You can use the filter to narrow your view, for example, to check that all the *Pods* have a status of `running`.
+. If the SMCP status indicates any problems, check the `status:` output in the YAML file for more information.
+
+
+.From the command line
+
+. Execute the following command to see if the control plane pods are available and running, where `istio-system` is the namespace where you installed the SMCP.
++
+[source,terminal]
+----
+$ oc get pods -n istio-system
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                    READY   STATUS    RESTARTS   AGE
+grafana-6c47888749-dsztv                2/2     Running   0          37s
+istio-egressgateway-85fdc5b466-dgqgt    1/1     Running   0          36s
+istio-ingressgateway-844f785b79-pxbvb   1/1     Running   0          37s
+istiod-basic-c89b5b4bb-5jh8b            1/1     Running   0          104s
+jaeger-6ff889f874-rz2nm                 2/2     Running   0          34s
+prometheus-578df79589-p7p9k             3/3     Running   0          69s
+----
++
+. Check the status of the control plane deployment with the following command, where `istio-system` is the namespace where you deployed the SMCP.
++
+[source,terminal]
+----
+$ oc get smcp -n <istio-system>
+----
++
+The installation has finished successfully when the STATUS column is `ComponentsReady`.
++
+.Example output
+[source,terminal]
+----
+NAME    READY   STATUS            PROFILES      VERSION   AGE
+basic   9/9     ComponentsReady   ["default"]   2.0.1.1   19m
+----
+
++
+If you have modified and redeployed your control plane, the status should read `UpdateSuccessful`.
++
+.Example output
+[source,terminal]
+----
+NAME            READY     STATUS             TEMPLATE   VERSION   AGE
+basic-install   9/9       UpdateSuccessful   default               v1.1          3d16h
+----
++
+. If the SMCP status indicates anything other than `ComponentsReady` check the ‘status:’ output in the SCMP resource for more information.
```

Comment:
```suggestion
. If the SMCP status indicates anything other than `ComponentsReady` check the `status:` output in the SCMP resource for more information.
```

Reply:
not rendering properly in the preview if it's supposed to be `code` formatting

Reply:
It's a copy/paste issue.  Fixed it.